src/api/catalog-search/util.ts

```diff
@@ -29,6 +31,17 @@ const getStrSort = (isAscending: boolean): SortFunction => {
   return (p1, p2) => p1.name.localeCompare(p2.name) * (isAscending ? 1 : -1);
 };
 
+const getDownloadsSort = (isAscending: boolean): SortFunction => {
+  return (p1, p2) => {
+    if (p1.downloads !== p2.downloads) {
+      return (p1.downloads - p2.downloads) * (isAscending ? 1 : -1);
+    } else {
+      // break ties by alphabetical
+      return p1.name.localeCompare(p2.name) * (isAscending ? -1 : 1);
```

Comment:
nitpick, but this could re-use the `getStrSort` helper in this file

```ts
// break ties by alphabetical
return getStrSort(isAscending)(p1, p2);
```